ui.frontend/README.md

```diff
@@ -74,6 +74,18 @@ The final inclusion of client libraries on the sites page is as follows:
 
 The above inclusion can of course be modified by updating the Page Policy and/or modifying the categories and embed properties of respective client libraries.
 
+### Customizing Component Styling
+#### React Components
+The `@adobe/aem-core-cif-react-components` package comes with a set of sample styles which implement the Venia design. You can add those styles by adding
+```css
+@import url('~@adobe/aem-core-cif-react-components/dist/main.css');
+```
+to your `main.scss` file in `/src/main/site`.
+
+To customize the styles of the components, we provide a CSS template file that comes with the `@adobe/aem-core-cif-react-components` package. It includes all classes that are used by the React components. The file is located in `/node_modules/@adobe/aem-core-cif-react-components/dist/css-api`.
```

Comment:
it's actually multiple files, right?

Reply:
Currently it's only one. The script handles basically all of them, in case would change the webpack configuration.